packages/workflow/src/workflow.ts

```diff
@@ -271,7 +308,12 @@ async function scheduleLocalActivityNextHandler({
  * Schedule an activity and run outbound interceptors
  * @hidden
  */
-export function scheduleActivity<R>(activityType: string, args: any[], options: ActivityOptions): Promise<R> {
+export function scheduleActivity<R>(
+  activityType: string,
+  args: any[],
+  options: ActivityOptions,
+  summary?: string
```

Comment:
Yeah… So, looking at this one, I really feel this should be part of `ActivityOptions`.